zipkin-ui/js/component_data/default.js

```diff
@@ -21,7 +21,7 @@ export default component(function DefaultData() {
     const query = convertToApiQuery(window.location.search);
     const serviceName = query.serviceName;
     if (serviceName) {
-      const apiURL = `/api/v1/traces?${queryString.stringify(query)}`;
+      const apiURL = `/zipkin/api/v1/traces?${queryString.stringify(query)}`;
```

Comment:
well at least not without significantly broadening the discussion beyond those interested in this UI (as it has deployment and other doc implications)